File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.py
# Last Updated: 2/5/2026, 9:21:58 AM
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        resultant=0
        for opr in operations:
            if opr in ["++X","X++"]:
                resultant+=1
            else:
                resultant-=1
        return resultant

File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.cpp
// Last Updated: 2/5/2026, 9:21:51 AM
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int result = 0 ; 
        for ( string opr : operations) result+=(opr[1] == '+' ? 1 : -1);
        return result;
    }
};